### ISSUE TITLE
Harmonize Right-Wing Violence Incident Reporting

### ISSUE THREAD

**Issue by jfilter** [codeforberlin/tickets#10]
Right now, there are several regional public-funded organization that monitor right-wing violence. Each  of them uses their own way of reporting an incident on their website.

Here is a list of all organizations: <https://verband-brg.de/index.php/ueber-uns/mitglieder>

And here a visualization: <https://vis.one/rechte-gewalt/>

tasks:
1. define a standard format
2. scrape & convert all incidents

recommended techonology:
* <https://morph.io/>

contact person: @jfilter

**Comment by kremio** [comment]
Hi!
I'd like to help with this. To start with I could go through the websites listed at https://verband-brg.de/index.php/ueber-uns/mitglieder and produce a list of where the reports are displayed and what information is available. For instance, the B.U.D. does not seem to have incidents reports on its site; ReachOut Berlin has them under https://www.reachoutberlin.de/de/chronik, with date of the incident, location (Kiez), a description of the incident and a list of other sources (police, newspapers) with the date of publication.

Would this be helpful?

**Comment by jfilter** [comment]
yes, definitely.

**Comment by kremio** [comment]
Hi!
I setup a Git project for this: https://github.com/kremio/right-wing-violence-de
and started surveying some sites on the list.
If anyone is interested in helping surveying the monitoring sites, please have a look at the issues.

**Comment by kremio** [comment]
Hi!

I went through all the sites and collected detailed information about what sort of data is available, and how it is embedded in the HTML. https://github.com/kremio/right-wing-violence-de/blob/master/docs/Topology.md

Before starting to write scrappers, it would be a good time to define the standard format. I created an issue for this task : https://github.com/kremio/right-wing-violence-de/issues/8

I'm happy to make a proposal, but this is definitely something that should be done in a group.
I can come to the meetup at the Wikimedia's office next monday if someone is interested in working on this.
